src/application_config/application_mode_debug.h

```diff
@@ -1,20 +1,43 @@
+/**
+ * @file application_config/debug.h
+ * @author Otso Jousimaa <otso@ojousima.net>
+ * @date 2020-04-16
+ * @brief  predefine  defaults in  app_config
+ * This is included by _modes followed by _default then _config
+ *
+ * @copyright Ruuvi Innovations Ltd, license BSD-3-Clause.
+ *
+ *  EDIT THIS file to customize
+ */ 
 #ifndef APPLICATION_MODE_DEBUG_H
 #define APPLICATION_MODE_DEBUG_H
 
 #define APP_FW_VARIANT "+debug"
-#define RI_LOG_ENABLED 1
+#define RI_LOG_ENABLED (true)
```

Comment:
(1U)

Reply:
booleans like xxxxxxx_ENABLED should be defined as (true).  Right?